telegram/ext/_applicationbuilder.py

```diff
@@ -198,7 +199,9 @@ def __init__(self: "InitApplicationBuilder"):
         self._context_types: DVType[ContextTypes] = DefaultValue(ContextTypes())
         self._application_class: DVType[Type[Application]] = DefaultValue(Application)
         self._application_kwargs: Dict[str, object] = {}
-        self._concurrent_updates: Union[int, DefaultValue[bool]] = DEFAULT_FALSE
+        self._concurrent_updates: Union[
+            int, DefaultValue[bool], "BaseUpdateProcessor"
+        ] = DEFAULT_FALSE
```

Comment:
```suggestion
        self._update_processor: "BaseUpdateProcessor" = SimpleUpdateProcessor(max_concurrent_updates=1)
```

If `concurrent_updates` in not called, then this should be the default value that we'd like to user, right? :)